### ISSUE TITLE
No clear separation between Nix and NixOS anymore

### ISSUE THREAD

**Issue by andir** [NixOS/nixos-homepage#467]
Since a few days (or weeks?) the website now lists Nix and NixOS side-by-side to each other. In the past I commonly explicitly linked people to either Nix or NixOS depending on the context we were having the conversation. It is fairly confusing for Non-Nix-People to distinguish between it even after a few conversations. We shouldn't make this any harder.

A personal points about how the two projects are being presented now:
I think it is pretty hard to follow the flow of the website with this new newspaper-style about page (that is now called Features but still says `About`):

https://nixos.org/features.html

![image](https://user-images.githubusercontent.com/638836/84498351-ceaeb880-acb0-11ea-9d10-a98047496cad.png)

This isn't some kind of (new)paper and at least to me it is very distracting to read the text in this format.

**Comment by garbas** [comment]
This was discussed multiple times at marketing meetings. While I don't expect everybody to watch all those youtube videos, I'm trying to make a point that we considered this.

We don't try to present different projects, but concepts and then the line between Nix and NixOS project becomes blurry. The Features page is - for now - a copy of previous About pages from Nix and NixOS. Plan is to be reworked since a lot of concepts there a duplicating. But one step at the time and going forward with small steps, but always going forward.

**Comment by garbas** [comment]
I'm closing this since I'm not sure what actionable item to take out of this.
